ansible_collections/arista/avd/docs/requirements.txt

```diff
@@ -3,7 +3,7 @@ mkdocs-material<10
 mkdocs-material-extensions
 pymdown-extensions
 mdx_truly_sane_lists
-mike==1.1.2
+mike==2.0.0
```

Comment:
If we move to 2, we should release the version and just take the newest like we do with the remaining things.